en/08_Changelogs/6.0.0.md

```diff
@@ -679,6 +688,18 @@ This has been changed to use the [`PasswordStrength` constraint](https://symfony
 
 You can change the level of entropy required by passing one of the valid [minScore values](https://symfony.com/doc/current/reference/constraints/PasswordStrength.html#minscore) into [`ConfirmedPasswordField::setMinPasswordStrength()`](api:SilverStripe\Forms\ConfirmedPasswordField::setMinPasswordStrength()).
 
+### Status flags in the CMS
+
+In CMS 5 the CMS showed status flags for the [`SiteTree`](api:SilverStripe\CMS\Model\SiteTree) class inside the `/admin/pages` section, and occasionally for other models in grid fields - but this was inconsistent and was done in a variety of different ways depending on what was adding the flags and where they were displayed.
+
+We've standardised this in the new [`ModelData::getStatusFlags()`](api:SilverStripe\Model\ModelData::getStatusFlags()) method to define the flags, and [`ModelData::getStatusFlagMarkup()`](api:SilverStripe\Model\ModelData::getStatusFlagMarkup()) to build the HTML markup for them. This means that status flags can be displayed for *any* model in the CMS.
+
+This is already used to show what locale the data is in for models localised using [`tractorcow/silverstripe-fluent`](https://github.com/tractorcow-farm/silverstripe-fluent/), and what versioned stage it's in for models using the [`Versioned`](api:SilverStripe\Versioned\Versioned) extension.
+
+Status flags are displayed in breadcrumbs at the top of edit forms in the CMS, in the site tree for `CMSMain`, for each row in a grid field, and in [``]() and [``]().
```

Comment:
> and in [``]() and [``]()

Looks like you've forgotten to fill in some text and links here

Reply:
Oops, will fix this.

Reply:
Updated